emily/handler/src/api/handlers.rs

```diff
@@ -0,0 +1,82 @@
+//! Handlers for the blocklist client API
+
+use crate::client::risk_client;
+use crate::common::error::{Error, ErrorResponse};
+use crate::config::RiskAnalysisConfig;
+use reqwest::Client;
+use std::convert::Infallible;
+use tracing::error;
+use warp::{http::StatusCode, Rejection, Reply};
+
+/// Handles requests to check the blocklist status of a given address.
+/// Converts successful blocklist status results to JSON and returns them,
+/// or converts errors into Warp rejections.
+#[utoipa::path(
+    get,
+    operation_id = "checkAddress",
+    path = "/screen/{address}",
+    tag = "address",
+    params(
+    ("address" = String, Path, description = "Address to get risk assessment for")
+    ),
+    responses(
+    (status = 200, description = "Risk assessment retrieved successfully", body = BlocklistStatus),
+    (status = 400, description = "Invalid request body"),
+    (status = 404, description = "Address not found"),
+    (status = 405, description = "Method not allowed"),
+    (status = 500, description = "Internal server error")
+    )
+)]
+pub async fn check_address_handler(
```

Comment:
I feel like there's a lot of unnecessary boilerplate included from the `blocklist_client` when it seems like this attribute is the only thing you seem to need for your purposes. Or do you need to also be able to run this to validate the approach?

Reply:
No, but I'd rather merge this in with this extra stuff here so that each change is a single logical step. Here the logical step is "replace emily with the blocklist client and use it's autogeneration approach".